### PR TITLE
Improve readability of scheduling failure logs

### PR DESCRIPTION
**Pull request by karlkfi**
- Add struct fields to log output.

### CODE REVIEW

**Comment by sttts** [comment]
Great idea. lgtm.

File: contrib/mesos/pkg/scheduler/plugin.go [hunk 1]
```diff
@@ -754,10 +754,10 @@ func (s *schedulingPlugin) Run(done <-chan struct{}) {
 // with the Modeler stuff removed since we don't use it because we have mesos.
 func (s *schedulingPlugin) scheduleOne() {
 	pod := s.config.NextPod()
-	log.V(3).Infof("Attempting to schedule: %v", pod)
+	log.V(3).Infof("Attempting to schedule: %+v", pod)
 	dest, err := s.config.Algorithm.Schedule(pod, s.config.MinionLister) // call kubeScheduler.Schedule
 	if err != nil {
-		log.V(1).Infof("Failed to schedule: %v", pod)
+		log.V(1).Infof("Failed to schedule: %+v", pod)
```
Comment (sttts) [inline contrib/mesos/pkg/scheduler/plugin.go]: Why do we drop the err here in the log?

Comment (karlkfi) [inline contrib/mesos/pkg/scheduler/plugin.go]: I think the error bubbles up to a log a few lines later.

**Comment by karlkfi** [comment]
@davidopp Pretty minimal change here. You wanna give it a thumb?

**Comment by davidopp** [comment]
LGTM

**Comment by gmarek** [comment]
Please rebase.

**Comment by sttts** [comment]
Rebased.

**Pull request merged**